### PR TITLE
Add validations for Stabilizationwindowseconds

### PR DESCRIPTION
**Pull request by Indresh2410**
_Provide a description of what has been changed_

### Checklist

- [x] When introducing a new scaler, I agree with the [scaling governance policy](https://github.com/kedacore/governance/blob/main/SCALERS.md)
- [x] I have verified that my change is according to the [deprecations & breaking changes policy](https://github.com/kedacore/governance/blob/main/DEPRECATIONS.md)
- [x] Tests have been added
- [x] Changelog has been updated and is aligned with our [changelog requirements](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#Changelog)
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))

Fixes https://github.com/kedacore/keda/issues/4976

Relates to #

### CODE REVIEW

File: config/crd/bases/keda.sh_scaledobjects.yaml [hunk 1]
```diff
@@ -138,6 +138,8 @@ spec:
                                   - For scale down: 300 (i.e. the stabilization window
                                   is 300 seconds long).'
                                 format: int32
+                                maximum: 3600
```
Comment (zroubalik) [inline config/crd/bases/keda.sh_scaledobjects.yaml:141]: We shouldn't hardcode values in here, even through `vendor`. Next deps update will remove your fix.


Could you please create a patch for kustomize instead? You can check existing ScaledJob patch for reference:
https://github.com/kedacore/keda/blob/main/config/crd/patches/scaledjob_patch.yaml
https://github.com/kedacore/keda/blob/b7c3d24cf37812998a2e2773d8823b7fb96b9331/config/crd/kustomization.yaml#L12

Comment (Indresh2410) [reply config/crd/bases/keda.sh_scaledobjects.yaml]: Thanks @zroubalik . Have addressed them

**Review by JorTurFer — APPROVED** [review]

**Review by zroubalik — APPROVED** [review]
LGTM, thanks for the contribution @Indresh2410!

**Pull request merged**